jni/include/faiss_wrapper.h

```diff
@@ -29,6 +29,12 @@ namespace knn_jni {
                                      jlong vectorsAddressJ, jint dimJ, jstring indexPathJ, jbyteArray templateIndexJ,
                                      jobject parametersJ);
 
+        // Create an index with ids and vectors. Instead of creating a new index, this function creates the index
+        // based off of the template index passed in. The index is serialized to indexPathJ.
+        void CreateBinaryIndexFromTemplate(knn_jni::JNIUtilInterface * jniUtil, JNIEnv * env, jintArray idsJ,
```

Comment:
Why introduce <Operation>Binary<...> methods, but for Free take parameter?

Reply:
It is to reflect underlying cpp method.

Faiss has two different method between non binary and binary for create index method. However, there is only one method, delete` to free the memory.